### ISSUE TITLE
derive: implement #[valuable] attributes

### ISSUE THREAD

**Issue by taiki-e** [tokio-rs/valuable#51]
- [x] `#[valuable(rename = "...")]`; #75
- [x] `#[valuable(transparent)]`; #75
- [x] `#[valuable(skip)]`; #75

**Comment by taiki-e** [comment]
As a first step, I'm planning to implement attributes like `rename`, `skip` that have non-controversial behavior and are relatively simple.

**Comment by Larandar** [comment]
This might be less trivial than it sounds, but another non-controversial would also be `#[valuable(redacted)]`. Almost like `skip` but semantically different.

**Comment by taiki-e** [comment]
I don't think we should consider that an attribute that an equivalent is no available in serde is "non-controversial" here. I cannot easily imagine how it would work.

**Comment by taiki-e** [comment]
The basic ones have been implemented in https://github.com/tokio-rs/valuable/pull/75. If someone has a request for other attributes, please open a separate issue.
